### DISCUSSION TITLE
problem about point source

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#743] [Q&A] [answered]
I run this example(https://github.com/buzhiqiuyuebai/Error-with-Detected-Inf-NaN-volume), at time `0.0041S` the rusult seems to be right, but at time `0.0085S` it seems to be wrong. I am not sure if it's a  problem about `point source`.
![image](https://user-images.githubusercontent.com/114474894/204127131-36ca858c-bf57-4c6e-88e2-3c6347894f93.png)
![image](https://user-images.githubusercontent.com/114474894/204127178-db4b8364-3f59-4017-a772-ae95c4c54bd0.png)
![image](https://user-images.githubusercontent.com/114474894/204127230-5c4273b8-7021-439c-991c-021fe29bbb97.png)

May I ask if I use the parameters (https://github.com/SeisSol/Examples/blob/master/WP2_LOH3/parameters.par) by justify the parameter `FreqCentral=2.5,FreqRatio=100 ` to get the result like this or change other parameters. His waveform looks complete, and the red and blue blocks of the signal in the middle only take up a small part, and the waveform can be seen clearly on the edge. My result is only correct at` 0.0041 seconds`, and it seems very bad.
![image](https://user-images.githubusercontent.com/114474894/204127683-582d9c9f-7b81-4a4a-b908-60a3e168b6da.png)

**Comment by sebwolf-de** [comment]
Hi, may I ask, why the result seems to be wrong? For me, the result looks reasonable. IMHO, you should rescale the color bar in paraview and then everything should be as expected.

**Reply by buzhiqiuyuebai** [reply]
> Hi, may I ask, why the result seems to be wrong? For me, the result looks reasonable. IMHO, you should rescale the color bar in paraview and then everything should be as expected.

What I'm not quite sure about is why at `0.0085` seconds the blue-red block in the middle is so big, which doesn't look the same as the previous example WP2_LOH3.

**Comment by Thomas-Ulrich** [comment]
I think the problem is 
```
FixTimeStep = 5
```
See https://github.com/SeisSol/SeisSol/pull/404 if you need more details.

**Comment by buzhiqiuyuebai** [comment, marked as answer]
I find the problem, The result is that my simulation time is too short, but the point source time is too long. I tried a new stimulus source and got the result I wanted.
![image](https://user-images.githubusercontent.com/114474894/205355716-16343c06-fd64-406d-a4fe-10569a7101bd.png)
![image](https://user-images.githubusercontent.com/114474894/205356399-f50f3f57-303b-4e8b-9483-884dc5d1cf2b.png)
